### PR TITLE
Doctype fix from 30

### PR DESCRIPTION
**Pull request by bdaisey**

### CODE REVIEW

**Comment by bdaisey** [comment]
@dcwatson this replaces https://github.com/imsweb/django-dynamic-attachments/pull/26 and addresses your feedback from there and from the [Squish issue](https://www.squishlist.com/ims/bioshare/1748/), namely it uses the existing 'data' field and is now a more simple fix.

Until the changes are merged into Bioshare 10.0, you can use [this branch](https://git.imsweb.com/bioshare/bioshare/tree/1748_fix_doctype_from_10) for testing.

Let me know if you have any questions!

File: attachments/models.py [hunk 1]
```diff
@@ -148,12 +148,11 @@ def __str__(self):
...
         return [ch.strip() for ch in self.choices.split('\n') if ch.strip()]
 
-    @property
-    def model_queryset(self):
+    def model_queryset(self, **kwargs):
         ModelClass = import_class(self.model)
         # Lookup models can provide an @classmethod 'field_model_queryset' to have control over what queryset is used
         if hasattr(ModelClass, 'field_model_queryset'):
-            qs = getattr(ModelClass, 'field_model_queryset')()
+            qs = getattr(ModelClass, 'field_model_queryset')(**kwargs)
```
Comment (tomlinsj) [inline attachments/models.py]: I think we want this wrapped in a try catch

most sites field_model_queryset functions probably dont accept kwargs so we should try with kwargs and without if it errors.

File: attachments/models.py [hunk 2]
```diff
@@ -148,12 +148,14 @@ def __str__(self):
...
-    def model_queryset(self):
+    def model_queryset(self, **kwargs):
         ModelClass = import_class(self.model)
         # Lookup models can provide an @classmethod 'field_model_queryset' to have control over what queryset is used
         if hasattr(ModelClass, 'field_model_queryset'):
-            qs = getattr(ModelClass, 'field_model_queryset')()
+            try:
+                qs = getattr(ModelClass, 'field_model_queryset')(**kwargs)
+            except:
+                qs = getattr(ModelClass, 'field_model_queryset')
```
Comment (tomlinsj) [inline attachments/models.py]: qs = getattr(ModelClass, 'field_model_queryset')()

Comment (bdaisey) [reply attachments/models.py]: Updated!  Tested and still works as expected.

**Pull request merged**